modules/Classroom/components/AssignmentCard/AssignmentCard.tsx

```diff
@@ -84,6 +88,15 @@ const AssignmentCard: React.FC<IAssignmentCardProps> = ({ assignment, classroomI
         isOpen={deleteModalOpened}
         onClose={() => setDeleteModalOpened(false)}
       />
+      {user?.userId && (
```

Comment:
why this guard is required?

Reply:
Bhaiya, there was an error like user.userId could also be null, and the SubmitAssignmentModal is expecting an userId with type number.